quarkus/admin/build.gradle.kts

```diff
@@ -72,28 +82,48 @@ quarkus {
   )
 }
 
-publishing {
-  publications {
-    named<MavenPublication>("maven") {
-      val quarkusBuild = tasks.getByName<QuarkusBuild>("quarkusBuild")
-      artifact(quarkusBuild.runnerJar) {
-        classifier = "runner"
-        builtBy(quarkusBuild)
-      }
+distributions {
+  main {
+    contents {
+      from(runScript)
+      from(project.layout.buildDirectory.dir("quarkus-app"))
+      from("../../NOTICE")
```

Comment:
IMHO it's better to square the license/notice on the target artifacts. I can change that in second step along with license/notice.